draft-ietf-acme-acme.md

```diff
@@ -2547,12 +2547,12 @@ connections to the ACME server and the validation channel outbound HTTP or DNS
 requests.
 
 Broadly speaking, ACME aims to be secure against active and passive attackers on
-any individual channel.  Some vulnerabilities arise (noted below), when an
+any individual channel.  Some vulnerabilities arise (noted below) when an
 attacker can exploit both the ACME channel and one of the others.
 
 On the ACME channel, in addition to network-layer attackers, we also need to
-account for application-layer man in the middle attacks, and for abusive use of
-the protocol itself.  Protection against application-layer MitM addresses
+account for man-in-the-middle attacks at the application-layer, and for abusive
```

Comment:
I think it might have been happier if the document was consistent. I'm not sure.